templates/general.js

```diff
@@ -85,17 +85,29 @@ ${name};
     return `
 {
   "name": "${moduleName}",
+  "title": "${moduleName.split('-').map(word => word[0].toUpperCase() + word.substr(1)).join(' ')}",
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
   "scripts": {
     "test": "echo \\"Error: no test specified\\" && exit 1"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/${authorGithub}/${moduleName}.git",
```

Comment:
What if the url for the repository is at a different place as at the authors github username. Thinking off such libs that are under a organization.

Reply:
Yes, I considered that. Perhaps the naming should be changed to `githubAccount` or `hostingGithubAccount`. 

I guess that brings up the next thing which is what if you use bitbucket or another service. I think this template / generator will serve most needs out of the box but in some cases a little modification may be necessary as it otherwise would make the options too complex.